app/main/lib/check_api.py

```diff
@@ -0,0 +1,14 @@
+import uuid
+from flask import current_app as app
+import requests
+
+class CheckAPI:
+    @staticmethod
+    def return_storage_webhook(check_api_host, action, model_type, data):
+        check_api_token = app.config['CHECK_API_TOKEN']
+        request = requests.post(f"{check_api_host}/api/webhooks/alegre?token={check_api_token}", json={
+            "action": action,
+            "model_type": model_type,
+            "data": data,
+        })
+        return request
```

Comment:
Per my comment above, it feels like this should be a URL that Check API passes to Alegre when calling the API and not something Alegre should come up with on its own. I'm fine with it as a short term measure, however.

Reply:
The only problem with doing that is that we have to persist and plumb the URL that Check API would pass through *every* request (alegre->presto, presto->alegre), or come up with a persistence mechanism. I think this is reasonable in the short term as, inside the data, there will always be an unambiguous ID that Check API-side can unpack to determine state, *and*, that URL that is being used won't ever change on the Check API side anyways (i.e. that is always going to be the endpoint that receives this sort of thing, no matter what)

Reply:
Agreed, but there are two longer term issues. 
(1) We ultimately need Alegre to work with Check API and Timpani. This means the callback URL will be different depending on the service calling Alegre. 
(2) Alegre shouldn't know anything about its callers. To date, Alegre has no knowledge/specifics about what Check API is or even that it exists. It feels like best practice for a microservice to keep Alegre agnostic to any specifics of its caller. This goes to the question that @caiosba asked about the controllers/constants for Check API.

Reply:
@DGaffney and I discussed that yesterday and I think we agreed to rename the constants/controller to something generic like "callback".